sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/models/PolicyResult.java

```diff
@@ -1,140 +1,32 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-// Code generated by Microsoft (R) AutoRest Code Generator.
 
 package com.azure.security.attestation.models;
 
-import com.azure.core.annotation.Fluent;
-import com.azure.core.util.Base64Url;
 import com.azure.core.util.BinaryData;
-import com.azure.core.util.CoreUtils;
-import com.fasterxml.jackson.annotation.JsonProperty;
-
-/** The result of a policy certificate modification. */
-@Fluent
-public final class PolicyResult {
-    /*
-     * The result of the operation
-     */
-    @JsonProperty(value = "x-ms-policy-result")
-    private PolicyModification policyResolution;
-
-    /*
-     * The SHA256 hash of the policy object modified
-     */
-    @JsonProperty(value = "x-ms-policy-token-hash")
-    private Base64Url policyTokenHash;
-
-    /*
-     * The certificate used to sign the policy object, if specified
-     */
-    @JsonProperty(value = "x-ms-policy-signer")
-    private JsonWebKey policySigner;
-
-    /*
-     * A JSON Web Token containing a StoredAttestationPolicy object with the
-     * attestation policy
-     */
-    @JsonProperty(value = "x-ms-policy")
-    private String policy;
 
+/**
+ * Represents the result of a policy modification operation (setAttestationPolicy/resetAttestationPolicy).
+ */
+public interface PolicyResult {
```

Comment:
>PolicyResult [](http://example.com/codeflow?start=17&length=12)

Should this be decorated with Immutable too?

Reply:
Yes. Fixing.